nx/lib/nx.ex

```diff
@@ -5244,6 +5244,39 @@ defmodule Nx do
     impl!(tensor).conjugate(%{tensor | type: Nx.Type.to_complex(tensor.type)}, tensor)
   end
 
+  @doc """
+  Calculates the complex phase angle of each element in the tensor.
+
+  If $z = a + bi = r e^\\theta$, $phase(z) = {\rm atan2}(b, a)$
```

Comment:
```suggestion
  If $z = a + bi = r e^\\theta$, $phase(z) = {\\rm atan2}(b, a)$
```

What does `\rm` do?

Reply:
`\rm` sets a fixed font: upright, roman, medium weight. I used it because there's no built in \atan2 or \arctan2.
Now that I searched about it, I found that it is deprecated in LaTeX, so I am going to remove it.